### ISSUE TITLE
DPL-265 - Additional deck validation on the Hamilton Heron Pooling protocol (C=?, V=5)

### ISSUE THREAD

**Issue by stevieing** [sanger/limber#991]
**User story**
The Heron team wants the Hamilton robot to dispense PhiX, instead of staff. I have been asked to program this protocol and will take 2-3 weeks to program, validate and implement it.
 
Sadly, after checking in with Heron, I have been told that these changes in the Hamilton protocol should reflect the LIMS as well. I understand that PhiX addition etc is already tracked in the LIMS, however to bring it in line with the rest of the deck, they want to include bed validation. I assume it’s just adding another deck location in Limber and cross referencing this with PhiX batches.

**Who are the primary contacts for this story**
Iraad

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by andrewsparkes** [comment]
What step is this, what is on the robot at this time? Is this an existing bed verification step that needs PhiX adding, or a new one?
What are the source and destination plate purposes, and what beds are they on.

Bed verification is set up to check the positions of plate/tube purposes and their states on robot beds, I doubt it is simple to start including PhiX tube verification as those tubes are not a pipeline plate/tube with a purpose.

**Comment by andrewsparkes** [comment]
From discussion with Iraad:
This is an existing bed verification, in Limber robots file it's called 'hamilton-lthr-384-lib-pcr-pool-to-lthr-384-pool-xp'.
This transfers/stamps multiple Lib PCR Pool plates into multiple Pool XP plates.
There will be one PhiX tube on the deck, in bed 28, used for all the destination plates (one tube contains enough for 6 pool plates).
PhiX tubes appear to be set up in advance in batches, so it should be possible to verify that the barcode scanned matches to those.
Iraad to check with Stef whether it's just a bed verification check they want, or if we need to track that use of PhiX batch number in the LIMS against the destination plate wells as well.

**Comment by andrewsparkes** [comment]
From Irfaan who makes the PhiX for the Heron team: the PhiX is registered in Sequencescape as a batch and a normalised batch. These will have barcodes that are generated in Sequencescape.
